### ISSUE TITLE
Enable parallel builds to speedup multi module builds

### ISSUE THREAD

**Issue by nmiodice** [Azure/iotedgedev#385]
As each module is a separate docker module they should all be able to be built in parallel. Adding this as a feature in the CLI will make our Azure Dev Ops builds an order of magnitude faster.

**Comment by blackchoey** [comment]
Hi @nmiodice , thanks for raising this great idea. According to moby issue [#9656](https://github.com/moby/moby/issues/9656), concurrent builds takes longer than sequential builds. Seems it has already been fixed but haven't been confirmed officially.
We will try to add parallel option to iotedgedev CLI and test the performance to see whether we would like to enable this feature. Please stay tuned for latest news.

**Comment by StefanSchoof** [comment]
https://github.com/moby/buildkit has a feature to build in parallel and also the possibility to cache. I would like to have an option to use buildkit.

**Comment by standemchuk** [comment]
@blackchoey hi, has there been any news on this front? Is there anything I can do to help speed things up?

**Comment by blackchoey** [comment]
@standemchuk Could you share how many modules you're going to build and how long did it take for sequential build and concurrent build?
As mentioned by moby issue [#9656](https://github.com/moby/moby/issues/9656), concurrent builds takes longer than sequential builds and the issue hasn't been resolved yet. So we need to be careful to avoid introducing performance issues. If you could help us understand the performance gains after enabled concurrent build with your real cases, that would be great.

**Comment by konichi3** [comment]
Hi 

Thank you for filing the issue.

We are looking to clean up the reported issues that are obsolete such that our team can focus on the latest issues reported.
As such, we are going to close the issue for now.

If the issue is still blocking or critical to you, please file a new issue and we will do our best to respond as soon as possible.
When you file a new issue, please ensure following additional information are provided.

o	If you believe it’s a bug, provide detail repro steps. 
o	If it’s a feature request, please note as such
o	Expected outcome
o	Is this a blocking issue?

Thank you,
Azure IoT Developer and Device team
